src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -510,70 +511,94 @@ def handle_list_tasks_command(
     db_session: Session,
 ) -> None:
     """Handles the list tasks command."""
-    blocks = []
+    tasks = task_service.get_all_by_incident_id(
+        db_session=db_session,
+        incident_id=context["subject"].id,
+    )
 
     caller_only = False
+    user_id = None
     if body["command"] == context["config"].slack_command_list_my_tasks:
+        user_id = payload["user_id"]
         caller_only = True
 
+    if caller_only:
+        user_id = user_id
```

Comment:
Isn't this a no-op? What if `user_id` is `None`, won't the `client.users_info()` function fail?

Reply:
This block of code was sloppy in general, addressed this issue, and cleaned it up.